app/web/src/components/AssetDetailIntrinsicInput.vue

```diff
@@ -117,7 +141,7 @@ watch(
   { immediate: true },
 );
 
-const emit = defineEmits(["change", "changeToUnset", "changeToIdentity"]);
+const emit = defineEmits(["change", "changeIntrinsicFunc"]);
```

Comment:
Simplify the emits. This probably could have been an additional one, but this PR originally had `resourcePayloadToValue` customization. Even though it doesn't now, I think this change is still for the better for the future.